encoding.bs

```diff
@@ -1109,17 +1109,25 @@ interface mixin TextDecoderCommon {
 };
 </pre>
 
-<p>The {{TextDecoderCommon}} interface mixin defines common attributes that are shared between
-{{TextDecoder}} and {{TextDecoderStream}} objects. These objects have an associated
-<dfn id=textdecoder-encoding for=TextDecoderCommon>encoding</dfn>,
-<dfn id=textdecoder-ignore-bom-flag for=TextDecoderCommon>ignore BOM</dfn> (initially false),
-<dfn id=textdecoder-bom-seen-flag for=TextDecoderCommon>BOM seen</dfn> (initially false), and
-<dfn id=textdecoder-error-mode for=TextDecoderCommon>error mode</dfn> (initially
-"<code>replacement</code>").
+<p>The {{TextDecoderCommon}} interface mixin defines common getters that are shared between
+{{TextDecoder}} and {{TextDecoderStream}} objects. These objects have an associated:
```

Comment:
Although I don't have a concrete idea as to why, "an associated" followed by a `<dl>` seems weird. May I suggest "have the following associated data" or something like that? I dunno.

Reply:
I'm going to leave it for now until I hear more complaints. I think it's only weird because we normally use them for switches and perhaps not always say so explicitly in the paragraph preceding one.